### ISSUE TITLE
Dev app doesn't work in IE

### ISSUE THREAD

**Issue by dmt0** [plotly/react-chart-editor#854]
`fetch` needs to be polyfilled

**Comment by VeraZab** [comment]
and i think the css is not working there too, we're compiling 2 versions of sass, one for IE specifically: https://github.com/plotly/react-chart-editor/blob/master/package.json#L114

**Comment by dmt0** [comment]
Yes, it's all those custom properties like `var(--sidebar-width)` - they're not supported in IE, our tools used to take care of those though.

**Comment by dmt0** [comment]
Were we ever able to test RCE in IE without embedding it into another project?

**Comment by VeraZab** [comment]
I always was testing from streambed..

**Comment by nicolaskruchten** [comment]
Did you see this part of the readme? https://github.com/plotly/react-chart-editor#styling-the-defaulteditor--and-the-built-in-components

If the dev/demo apps could work in IE that would be helpful IMO

**Comment by chl8725** [comment]
> Did you see this part of the readme? https://github.com/plotly/react-chart-editor#styling-the-defaulteditor--and-the-built-in-components
> 
> If the dev/demo apps could work in IE that would be helpful IMO

I can't display in ie.
But go to the link you said
I found import'react-chart-editor/lib/react-chart-editor.ie.min.css' but there is no react-chart-editor.ie.min.css file
where is the file?
